test/postgres-consistency/README.md

```diff
@@ -0,0 +1,16 @@
+# Postgres consistency tests
+
+## Overview
```

Comment:
Can you put some pointers to documentation that describes how the queries are generated, etc?

Alternatively, fold this workflow into `test/output-consistency` if there is enough commonality and extend the `README.md` in that directory.

Reply:
✅ Addressed in https://github.com/MaterializeInc/materialize/pull/21781/commits/4f71e7b234e3505ae43ee43b81e08af28dd72539.